src/css/components/_adaptive.scss

```diff
@@ -50,20 +50,18 @@
     }
   }
 
-  // Hide the subs-caps button for non-Live UI "x-small" and for "tiny" players.
-  &.vjs-layout-x-small:not(.vjs-liveui),
-  &.vjs-layout-x-small:not(.vjs-live),
+  // At x-small and tiny, the progress control is too narrow to be useful.
+  &.vjs-layout-x-small,
   &.vjs-layout-tiny {
-    .vjs-subs-caps-button {
+
+    .vjs-progress-control {
       display: none;
     }
   }
 
-  // With the new Live UI, we can have the same treatment as "tiny". At
-  // "x-small", the Live UI makes the progress control very small and almost
-  // useless.
-  &.vjs-layout-x-small.vjs-liveui,
-  &.vjs-layout-tiny {
+  // At x-small, the buttons alone leave a large gap on the right. Fill it with
+  // the spacer element.
+  &.vjs-layout-x-small {
```

Comment:
Even with the progress control hidden, there is still some real estate left in some cases at `x-small`, so instead of a large gap on the right, use the spacer element in the middle to fill the real estate.